core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3059,10 +3059,27 @@ private boolean syncMetadata(RpcContext rpcContext, LockedInodePath inodePath,
         // The requested path does not exist in Alluxio, so just load metadata.
         pathsToLoad.add(inodePath.getUri().getPath());
       } else {
-        SyncResult result =
-            syncInodeMetadata(rpcContext, inodePath, syncDescendantType);
-        deletedInode = result.getDeletedInode();
-        pathsToLoad = result.getPathsToLoad();
+        AlluxioURI path = inodePath.getUri();
```

Comment:
NIT: In this block, it is listing the entire ufs subtree, to "sync" it with the Alluxio subtree. We should add a brief comment on that.